runtime/src/tests/functionalTests/protractorTS.config.ts

```diff
@@ -21,10 +21,15 @@ let conf: Config = {
   // Ref: https://github.com/angular/protractor/tree/master/exampleTypescript/asyncAwait
   SELENIUM_PROMISE_MANAGER: false,
 
-  specs: ["specs/*.js", "specs/**/*.js"],
-
+  specs: [
+    "specs/smoke/smokeTest.spec.js", 
```

Comment:
Why do we have mention to .js files in TS config?

Reply:
@rgarg1 typescript compiler will generate .js file which you can run

Reply:
oh..okay. Thanks

Reply:
@rgarg1 now smokeTest are moved to a new folder smoke...So please delete smokeTest.spec.js inside functionalTests/specs/ and then try running.
smokeTest are moved to new folder to run smokeTest first in each PR and then other tests